client/a8c-for-agencies/components/user-contact-support-modal-form/index.tsx

```diff
@@ -199,39 +217,48 @@ export default function UserContactSupportModalForm( {
 					</FormSelect>
 				</FormFieldset>
 
-				<FormFieldset>
-					<FormLabel htmlFor="message">{ translate( 'How can we help?' ) }</FormLabel>
-					{ ! isPressableSelected ? (
-						<FormTextarea
-							name="message"
-							id="message"
-							placeholder={ translate( 'Add your message here' ) }
-							value={ message }
-							onChange={ onMessageChange }
-							onClick={ () =>
-								dispatch(
-									recordTracksEvent( 'calypso_a4a_user_contact_support_form_message_click' )
-								)
-							}
-						/>
-					) : (
-						<p>
+				{ isPressableSelected ? (
+					<>
+						<FormFieldset>
+							<FormLabel htmlFor="product">
+								{ translate( 'Would you like help with Pressable sales or support?' ) }
+							</FormLabel>
+							<FormSelect
+								name="pressable_contact"
+								id="product"
+								value={ pressableContactType }
+								onChange={ onPressableContactTypeChange }
+							>
+								<option value="sales">{ translate( 'Sales' ) }</option>
+								<option value="support">{ translate( 'Support' ) }</option>
+							</FormSelect>
+						</FormFieldset>
+						<div className="form-field-description">
 							{ translate(
-								'For help with Pressable, please {{a}}login to your Pressable account{{/a}} and request help using the chat widget on the bottom right of the page.',
-								{
-									components: {
-										a: (
-											<a
-												href="https://my.pressable.com/login"
-												target="_blank"
-												rel="noreferrer noopener"
-											/>
-										),
-									},
-								}
+								'Your request will be routed directly to a Pressable support specialist to chat about your needs.'
 							) }
-						</p>
-					) }
+						</div>
+					</>
+				) : null }
```

Comment:
Nitpicking: We can skip elvis operator here:

```

				{ isPressableSelected && (
					<>
						<FormFieldset>
							<FormLabel htmlFor="product">
								{ translate( 'Would you like help with Pressable sales or support?' ) }
							</FormLabel>
							<FormSelect
								name="pressable_contact"
								id="product"
								value={ pressableContactType }
								onChange={ onPressableContactTypeChange }
							>
								<option value="sales">{ translate( 'Sales' ) }</option>
								<option value="support">{ translate( 'Support' ) }</option>
							</FormSelect>
						</FormFieldset>
						<div className="form-field-description">
							{ translate(
								'Your request will be routed directly to a Pressable support specialist to chat about your needs.'
							) }
						</div>
					</>
				) }
```

Reply:
Yep! Thank you for pointing it out, much better.

Reply:
https://github.com/Automattic/wp-calypso/pull/94366/commits/1be27eb5317c89d4725c25eacd571b50221b4570